_schemaV2.graphql

```diff
@@ -6910,7 +6914,7 @@ type PartnerCategory {
     # term used for searching Partners
     term: String
     type: [PartnerClassification]
-  ): [Partner]
+  ): PartnerConnection
```

Comment:
This _might_ be a breaking change (to something that fetches `partner` nested under a partner category)?